packages/server/src/api/routes/tests/queries/mongodb.spec.ts

```diff
@@ -764,7 +764,7 @@ if (descriptions.length) {
         const verbs = ["read", "create", "update", "delete"]
         for (const verb of verbs) {
           const query = await createQuery({
-            fields: { json: {}, extra: { actionType: "invalid" } },
+            fields: { json: {}, extra: { actionType: "invalid" as any } },
```

Comment:
I _tend_ to prefer throwing a `// @ts-expect-error` in these cases because it makes it more explicit, you can explain you're testing an invalid value on purpose. Also at some point I'd love to start tracking the count of `any` in the codebase and aim toward 0.